repository/assets_maintenance_repo.go

```diff
@@ -0,0 +1,23 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/vkhichar/asset-management/domain"
+)
+
+type AssetMaintenanceRepo interface {
+	InsertMaintenance(ctx context.Context, assets_id string) (*domain.Maintenance, error)
```

Comment:
How will you be able to insert the maintenance activity by only providing id in the argument??

Also, Rename this method to : 'InsertMaintenanceActivity'